packages/core/src/textures/TextureSystem.js

```diff
@@ -39,6 +40,20 @@ export default class TextureSystem extends System
          * @readonly
          */
         this.managedTextures = [];
+
+        /**
+         * Did someone temper with textures state? We'll overwrite them when we need to unbind something.
+         * @member {boolean}
+         * @private
+         */
+        this._undefinedBoundTextures = false;
+
+        /**
+         * BaseTexture value that shows that we don't know what is bound
+         * @member {PIXI.BaseTexture}
+         * @readonly
+         */
+        this.undefinedTexture = new BaseTexture();
```

Comment:
can we just use `emptyTextures` here?

Reply:
We can do it like in v4, but it wont be clear that we use assignment in `reset()` to fake unknown bound textures and not to rebinding everything.